contact.js

```diff
@@ -0,0 +1,27 @@
+const form = document.getElementById('form');
+const email = document.getElementById('small-inputs');
+
+function isEmail(address) {
+  return address.match(/^[a-z]+@+.*$/);
+}
+
+function checkInputs() {
+  const emailValue = email.value.trim();
+
+  function setErrorFor(input, message) {
+    const formControl = input.parentElement;
+    const small = formControl.querySelector('small');
+    formControl.className = 'form-control error';
+    small.innerText = message;
+  }
+
+  if (!isEmail(emailValue)) {
+    setErrorFor(email, 'Can not submit email with uppercase characters');
+  } else { form.submit(); }
+}
+
+form.addEventListener('submit', (e) => {
+  e.preventDefault();
+
+  checkInputs();
+});
```

Comment:
There is an issue with email validation at the moment. If the uppercase is before the @, the error message is displayed correctly and the form is not submitted. Now, if the uppercase is after the @, it's being handled as a false negative and the form is submitted 😥 Kindly make sure that the validations are working on both cases 🚀 You got this!

Reply:
Thank you for noticing! Already made the correction.